hidden/Features/StatusBar/StatusBarController.swift

```diff
@@ -46,6 +50,15 @@ class StatusBarController{
         autoCollapseIfNeeded()
     }
     
+    private func setUpPermHideStatusBarIfNeeded() {
+        if Util.getEnablePermHide() {
+            permHideStatusBar = NSStatusBar.system.statusItem(withLength: 20)
+            if let button = permHideStatusBar?.button {
+                button.image = NSImage(named: NSImage.Name("ic_dot"))
+            }
+        }
+    }
```

Comment:
if Util.getEnablePermHide, let button = NSStatusBar.system.statusItem(withLength:20)?.button {
 button.image = NSImage(named"")
}
Reduce one level of if